README.md

```diff
@@ -1 +1,431 @@
-# Reference Guide: Objective-C Fundamentals
\ No newline at end of file
+# Reference Guide: Objective-C Fundamentals
+
+## Table of Contents
+
+* [NSLog Basics](#nslog-basics)
```

Comment:
(Unrelated thought: I wonder if internal links like this even work on learn.co. Worth testing when this is published.)